docs/src/pages/guides/DB2OnCloudSetup.md

```diff
@@ -116,7 +167,7 @@ Before the API key can be used, you need to create a Db2 user and associate it w
 
 1. Click on Open Console.
 
-1. Click `SETTINGS` > `Manage Users`.
+1. Click Administration > User Management
 
     - Note: If you do not see `Manage Users`, you are probably using a [non-IAM instance](https://www.ibm.com/support/knowledgecenter/SS6NHC/com.ibm.swg.im.dashdb.security.doc/doc/iam.html). To confirm:
```

Comment:
Can we remove this note now?  I'm not sure you can have a `Standard` or `Enterprise` plan that doesn't have IAM enabled

Reply:
If we do keep it, I guess we should change `Manage Users` to "this menu option" or some such

Reply:
paul's changes addressed this concern